packages/pelagos/src/listInput/ListEntries.js

```diff
@@ -52,26 +65,36 @@ const ListEntries = ({
 		return clearHighlight && clearHighlight.cancel;
 	}, [highlightKey, clearHighlight]);
 
+	const operationId = `${id}-operation`;
 	return (
 		<>
 			<div className="sr-only" aria-live="polite" ref={liveRef} />
+			{reorderable && (
+				<div id={operationId} className="sr-only">
+					{t`Press space bar to reorder`}
+				</div>
+			)}
 			<ul
+				ref={reorderRef}
 				id={id}
-				className={`ListEntries ListEntries--${column ? 'column' : 'grid'}${className ? ` ${className}` : ''}`}
+				className={`ListEntries ListEntries--${column || reorderable ? 'column' : 'grid'}${className ? ` ${className}` : ''}`}
 				onClick={handleClick}>
 				{list.map((item, i) => {
 					const name = getItemName(item);
 					const element = renderItem ? renderItem(item) : renderListItem(name);
-					let className = element.props.className;
-					className = className ? `${className} ListEntries__name` : 'ListEntries__name';
 					const itemKey = getItemKey(item, i);
 					return (
 						<Layer
 							key={itemKey}
 							as="li"
 							className={`ListEntries__item${itemKey === highlightKey ? ' ListEntries__item--highlight' : ''}`}
+							tabIndex={reorderable ? 0 : undefined}
+							aria-describedby={reorderable ? operationId : undefined}
 							data-testid="list-item">
-							{cloneElement(element, {className})}
```

Comment:
If the list is not reorderable, we should probably revert back to this `cloneElement` just to reduce the nesting in the DOM